service/tutorial/order/NewTutorialServices.xml

```diff
@@ -0,0 +1,165 @@
+
+    <service verb="get" noun="Persons">
+        <out-parameters>
+            <parameter name="personList" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.party.Person" list="personList">
+                <econdition field-name="maritalStatusEnumId" value="MarsMarried"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <!-- @3.Finding all Order Header records so that the latest placed order appears at the top-->
+
+    <service verb="get" noun="LatestOrders">
+        <out-parameters>
+            <parameter name="ordersList" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderHeader" list="ordersList">
+                <order-by field-name=" -placedDate"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <!-- 4.Finding the Order Header details for a given orderId.-->
+
+    <service verb="get" noun="Order">
+        <in-parameters>
+            <parameter name="orderId" required="true"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="ordersList" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderHeader" list="ordersList"/>
+            <set field="order" from="[]"/>
+            <iterate list="ordersList" entry="order">
+                <set field="orderId" from="order.orderId"/>
+                <set field="orderName" from="order.orderName"/>
+                <set field="placedDate" from="order.placedDate"/>
+                <set field="statusId" from="order.statusId"/>
+                <set field="orderRevision" from="order.orderRevision"/>
+                <set field="currencyUomId" from="order.currencyUomId"/>
+                <set field="productStoreId" from="order.productStoreId"/>
+                <set field="salesChannelEnumId" from="order.salesChannelEnumId"/>
+                <set field="grandTotal" from="order.grandTotal"/>
+                <set field="orderMap"
+                     from="[orderId:orderId,orderName:orderName,placedDate:placedDate,statusId:statusId,orderRevision:
+                     orderRevision,currencyUomId:currencyUomId,productStoreId:productStoreId,salesChannelEnumId:salesChannelEnumId
+                     ,grandTotal:grandTotal]"/>
+                <set field="order" from="order+orderMap"/>
+            </iterate>
+        </actions>
+    </service>
+
+    <!-- 5.Finding all Order Header records with the grandTotal greater than 50.-->
+
+    <service verb="get" noun="OrderHeaders">
+        <out-parameters>
+            <parameter name="orderHeaderDetails" type="list"/>
+        </out-parameters>
+        <actions>
+            <set field="orderHeaderDetails" from="[]"/>
+            <entity-find entity-name="mantle.order.OrderHeader" list="ordersList">
+            </entity-find>
+            <iterate list="ordersList" entry="order">
```

Comment:
Can we not put a condition in entity find itself. 
We should be returning all the details for the order header.
Please check.